microsetta_interface/tests/test_implementation.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, status_code, text, json_val, headers=None):
 
     def test_authrocket_callback_noargs_error(self):
         # We are observing exceptions on /authrocket_callback which occur with
-        # malformed requests. Currently, we are throwing a 500, but instead
+        # malformed requests. Currently,we are throwing a 500, but instead
```

Comment:
Can you explain why you removed the space from this comment?

Reply:
no reason, it must be a typo. It is not a technically thought change